script/utils/GenerateConstantsJson.ts

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env ts-node
+
+import * as fs from "fs";
+import * as path from "path";
+
+// Import the constants from the TypeScript files
+import { CHAIN_IDs } from "../../utils/constants";
+import {
+  ZERO_ADDRESS,
+  USDC,
+  USDCe,
+  WETH,
+  WGHO,
+  QUOTE_TIME_BUFFER,
+  FILL_DEADLINE_BUFFER,
+  ARBITRUM_MAX_SUBMISSION_COST,
+  AZERO_GAS_PRICE,
```

Comment:
We can probably drop references to Aleph Zero because the Aleph Zero deployment is EoL now.